soh/soh/Enhancements/randomizer/3drando/item_location.cpp

```diff
@@ -24,11 +24,11 @@ void LocationTable_Init() {
 
     //Lost Woods
     locationTable[LW_NEAR_SHORTCUTS_GROTTO_CHEST]        = ItemLocation::Chest      (RC_LW_NEAR_SHORTCUTS_GROTTO_CHEST,             0x3E, 0x14, "LW Near Shortcuts Grotto Chest",       LW_NEAR_SHORTCUTS_GROTTO_CHEST,        BLUE_RUPEE,                {},                                                                                                    SpoilerCollectionCheckGroup::GROUP_LOST_WOODS);
-    locationTable[LW_SKULL_KID]                          = ItemLocation::Base       (RC_LW_SKULL_KID,                               0x5B,       "LW Skull Kid",                         LW_SKULL_KID,                          PIECE_OF_HEART,            {},                                                   SpoilerCollectionCheck::ItemGetInf(30),          SpoilerCollectionCheckGroup::GROUP_LOST_WOODS);
+    locationTable[LW_SKULL_KID]                          = ItemLocation::Base       (RC_LW_SKULL_KID,                               0x5B,       "LW Skull Kid",                         LW_SKULL_KID,                          PIECE_OF_HEART,            {},                                                   SpoilerCollectionCheck::ItemGetInf(22),          SpoilerCollectionCheckGroup::GROUP_LOST_WOODS);
```

Comment:
Not really for this PR, but is this on the radar for Rando V3 to get enums for these?

Reply:
I do want to clarify on this, though, it would be useful to still keep track of these, as they're useful for non-rando tracking, if the randomizer infs aren't going to be flipped both in and out of rando.

Reply:
Wasn't the plan to move back to vanilla flags as much as possible? Anyway, yeah not really a conversation to hold here, just saw it and figured I'd mention it.